cmd/fortifyExecuteScan.go

```diff
@@ -957,16 +957,22 @@ func determinePullRequestMerge(config fortifyExecuteScanOptions) (string, string
 	if matches != nil && len(matches) > 1 {
 		return string(matches[config.PullRequestMessageRegexGroup]), author
 	}
-	return "", ""
+	return "0", ""
 }
 
 func determinePullRequestMergeGithub(ctx context.Context, config fortifyExecuteScanOptions, pullRequestServiceInstance pullRequestService) (string, string, error) {
+	number := "0"
+	email := ""
 	options := github.PullRequestListOptions{State: "closed", Sort: "updated", Direction: "desc"}
 	prList, _, err := pullRequestServiceInstance.ListPullRequestsWithCommit(ctx, config.Owner, config.Repository, config.CommitID, &options)
```

Comment:
Could it be that the response from ListPullRequestsWithCommit is empty/not present and no error is thrown?

```
	resp, err := s.client.Do(ctx, req, &pulls)
	if err != nil {
		return nil, resp, err
	}

	return pulls, resp, nil
```